eng/psakefile.ps1

```diff
@@ -114,6 +117,24 @@ task cargo-clippy -depends init {
     }
 }
 
+task checkmypy -depends wheelhouse {
+
+    # - Run mypy from within new venv. Reuse same script from task docs
+    Write-Host (Get-ChildItem $wheelhouse -Include *.whl)
+    $envPath = Join-Path $repo.root ".mypy-venv"
+    Create-PyEnv `
+        -EnvironmentPath $envPath `
+        -RequirementsPath (Join-Path $repo.root "eng" "lint-requirements.txt") `
+        -ArtifactPaths (Get-Item $wheelhouse)
+    & (Join-Path $envPath "bin" "Activate.ps1")
+    try {
+        mypy "$($pyqir.parser.python_dir)" "$($pyqir.generator.python_dir)" "$($pyqir.evaluator.python_dir)"
```

Comment:
In order for the build to fail, we have to check the exit code of `mypy`. `Invoke-LoggedCommand` builds on top of `exec` which will do this for us.
```suggestion
        Invoke-LoggedCommand -errorMessage "Please fix the above mypy errors" {
            mypy "$($pyqir.parser.python_dir)" "$($pyqir.generator.python_dir)" "$($pyqir.evaluator.python_dir)"
        }
```